html-relative-links/about-me/about-me.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>About Me</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href = "../index.html">Home Page</a></li>
+    <li><a href = "../about-me/about-me.html">About Me</a></li>
```

Comment:
Since we're already in the `about-me` folder, we can just do `<a href="about-me.html">`